### ISSUE TITLE
zsh: command not found: mike

### ISSUE THREAD

**Issue by timgavin** [jimporter/mike#141]
I've installed mike per the instructions (macOS), yet I kept receiving this error

```
zsh: command not found: mike
```

So I added the packages directory to my PATH...
```
export PATH="${HOME}/Library/Python/3.9/lib/python/site-packages:$PATH"
```

and now when I run `mike -V` get `zsh: permission denied: mike`

Any ideas how to get mike working?

**Comment by jimporter** [comment]
Mike just uses the normal setuptools methods for installing the command-line scripts, so all it should take is for you to install it via the usual methods (probably `pip install`). The [CI for mike](https://github.com/jimporter/mike/actions) verifies that this works on macOS; unfortunately, I don't use macOS myself, so I'm not able to provide any specific help here.

**Comment by timgavin** [comment]
I figured this out, and am leaving a comment in case anyone else has this issue.

macOS ships with Python 2, so I had previously installed Python 3 via Homebrew and forgot. This means that Python is now available at `$ python3` instead of `$ python`. Same basically goes for Pip. So I uninstalled mike, then ran the following

```
$ pip3 install mike
```

I then had to update my PATH with the location to Python 3 in `~/.zshrc`

```
export PATH="${HOME}/Library/Python/3.9/bin:$PATH"
```

Save the changes and close the file, then run `$ source` or start a new terminal session.

At this point I was able to run `$ mike serve` 👍
